Repair check of logbindir

qha opened the pull request:
Presumably the intention was to check that it is *both* not relative
and not already managed.

After update to 10.8 the mysql module tries and fails to manage '.'
on nodes where i have log-bin set to a relative path.

gnif commented:
I can confirm this issue is valid and the fix is correct. I just hit the identical issue.

qha commented:
I note that checks fail, i would be willing to put some time into repairing them if someone would provide some starting pointers.

david22swan commented:
Closing and re-opening pr in order to force the rekick of the tests. Some of the failures look like they may come from travis

david22swan commented:
@qha Taking a quick look at the failures, quite a few seem to come from some missing changes.
Could you rebase your changes onto the main, that should clear some of them up and make diagnosis easier.

qha commented:
Rebased, hope that does the trick. Thank you for your time and advice! @david22swan

qha commented:
Finally figured out at least one reason the tests were failing that was actually a bug (i don't think i introduced it, just changed things up so the tests caught it).

The remaining errors seem to be test environment problems if i understand the output...

david22swan commented:
@qha Thanks for responding so fast with the changes 👍 
I did a few re-kicks of the tests and their all passing now so I'm happy to merge

qha commented:
Thanks for merging! Do you know when a new release might happen? @david22swan

qha commented:
I see that a new version has been released now.

pmcmaw commented:
Yes apologies, this module has been released as of today and is now available on the Forge! 
Thank you for your patience :-) @qha
